qadence_protocols/mitigations/twirl.py

```diff
@@ -67,23 +60,53 @@ def mitigate(
     samples_twirl_den_list = []
     for twirl in twirls:
         layer = [X(i) for i in twirl]
-        block_twirl = chain(circuit.block, kron(*layer))
+        block_twirl = chain(block, kron(*layer))
 
         circ_twirl_num = QuantumCircuit(block_twirl.n_qubits, block_twirl)
-        model_twirl_num = QuantumModel(circuit=circ_twirl_num, backend=backend)
-        samples_twirl_num = model_twirl_num.sample(noise=noise, n_shots=n_shots)[0]
-        samples_twirl_num_list.append(twirl_swap(n_qubits, twirl, samples_twirl_num))
+        model_twirl_num = QuantumModel(circuit=circ_twirl_num, backend=model.backend.backend.name)
```

Comment:
Is this working ? 

I thought the syntax would be something like:

```suggestion
        model_twirl_num = QuantumModel(circuit=circ_twirl_num, backend=model._backend_name)
```

Reply:
Both work, but i changed it in accordance to your suggestion